bootstrap/app_test.go

```diff
@@ -628,7 +629,7 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
-	temporarySecretDirectory, err = os.MkdirTemp(".", ".secrets")
+	temporarySecretDirectory, err = os.MkdirTemp(os.TempDir(), ".secrets")
```

Comment:
this avoids the files popping up inside the bootstrap directory, when you e.g. debug a test and then quit debugging.